docs/configuration.md

```diff
@@ -0,0 +1,14 @@
+# Configuration
+
+## Logging
+
+By default, you cannot capture all pgvecto.rs logs. pgvecto.rs starts a background worker process for indexing, and it prints logs to standard error. To capture them, you need to set `logging_collector` to `on`. You can get more information from [PostgreSQL document about logging collector](https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOGGING-COLLECTOR).
+
+You can set `logging_collector` to `on` with the following command:
```

Comment:
This helps a lot. I am nowhere to found a log before, there might be a lot of user like me.